scripts/check_frontend_coverage.py

```diff
@@ -367,7 +367,6 @@
     'topic-viewer-domain.constants.ajs.ts',
     'topic-viewer-domain.constants.ts',
     'TopicRightsObjectFactory.ts',
-    'topics-and-skills-dashboard-domain.constants.ajs.ts',
```

Comment:
This file should not be removed. You should write tests to cover this.

Reply:
Let's discuss this in a separate [comment thread](https://github.com/oppia/oppia/pull/8856#pullrequestreview-377869438).